src/Dapr.Actors/DaprHttpInteractor.cs

```diff
@@ -57,7 +57,7 @@ internal class DaprHttpInteractor : IDaprInteractor
             this.httpClient.Timeout = requestTimeout ?? this.httpClient.Timeout;
         }
 
-        public async Task<string> GetStateAsync(string actorType, string actorId, string keyName, CancellationToken cancellationToken = default)
+        public async Task<ActorStateResponse<string>> GetStateAsync(string actorType, string actorId, string keyName, CancellationToken cancellationToken = default)
```

Comment:
We want to keep `ActorStateResponse` as an internal type. How do we keep complication happy with this constraint?

Reply:
@philliphoff any ideas here?

Reply:
`ActorStateResponse` is already `internal`, no, or are you referring to the fact that this method is `public`? `IDaprHttpInteractor` and `DaprrHttpInteractor` are already `internal` types, so nothing of them can be seen outside the SDK.  The `public` on the method only applies to other types within the SDK.  (Method visibility is always masked by visibility of the underlying type.)